src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Hover/DefaultRazorHoverInfoServiceTest.cs

```diff
@@ -445,8 +445,8 @@ public void GetHoverInfo_TagHelper_Element_VSClient_ReturnVSHover()
             var vsHover = service.GetHoverInfo(codeDocument, location, clientCapabilities);
 
             // Assert
-            Assert.False(vsHover.Contents.TryGetThird(out var _));
-            Assert.True(vsHover.Contents.TryGetSecond(out var _) && !vsHover.Contents.Second.Any());
+            Assert.False(vsHover.Contents.TryGetFourth(out var _));
+            Assert.True(vsHover.Contents.TryGetThird(out var _) && !vsHover.Contents.Third.Any());
```

Comment:
I'm not clear on why this changed.. if the sumtype changed, then shouldn't some definition in `RazorHoverEndpoint` or `RazorHoverInfoService` also change?

Reply:
The trick is that SumTypes will do an implicit conversion on the way in (MarkupContent becomes a SumType<string, MarkupContent>) but that can't happen on the way out.